images/configmap-reload/README.md

```diff
@@ -0,0 +1,43 @@
+<!--monopod:start-->
+# configmap-reload
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/configmap-reload` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/configmap-reload/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Kubernetes ConfigMap Reload
+
+`configmap-reload` is a simple binary to trigger a reload when Kubernetes ConfigMaps or Secrets, mounted into pods, are updated.
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/configmap-reload:latest
+```
+
+## Usage
+
+This image is a drop-in replacement for the upstream image.
+You can run it using the helm chart with:
+
+```shell
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+$ helm install my-release prometheus-community/alertmanager \
+    --set configmapReload.enabled=true \
+    --set configmapReload.image=cgr.dev/chainguard/configmap-reload \
+    --set image.tag=latest
+    <other configuration parameters here>
+```
+
+Note that the `configmap-reload` does need cloud provider configuration to work correctly, so it won't run locally.
```

Comment:
still refers the autoscaler and doesn't need  cloud provider here

Reply:
🤦